workbase/wrappers-bazel/npm_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+export PATH=`pwd`/external/nodejs/bin:$PATH
+export NODE_PATH=`pwd`/workbase/node_modules/
+
+pushd workbase
```

Comment:
Same question as the above: What happens when you run `bazel test` not from the root workspace directory? will this script still work?

Reply:
@haikalpribadi similarly, yes, it will work